server/models/video/video-channel-sync.ts

```diff
@@ -0,0 +1,177 @@
+import { isVideoChannelSyncStateValid } from "@server/helpers/custom-validators/video-channel-syncs"
+import { CONSTRAINTS_FIELDS, VIDEO_CHANNEL_SYNC_STATE } from "@server/initializers/constants"
+import { VideoChannelSync, VideoChannelSyncState } from "@shared/models"
+import { MChannelSyncChannel, MChannelSyncFormattable } from "@server/types/models/video/video-channel-sync"
+import { AttributesOnly } from "@shared/typescript-utils"
+import {
+  AllowNull,
+  BelongsTo,
+  Column,
+  CreatedAt,
+  DataType,
+  Default,
+  DefaultScope,
+  ForeignKey,
+  Is,
+  Model,
+  Table,
+  UpdatedAt
+} from "sequelize-typescript"
+import { AccountModel } from "../account/account"
+import { getSort, throwIfNotValid } from "../utils"
+import { VideoChannelModel } from "./video-channel"
+import { Op } from "sequelize"
+import { UserModel } from "../user/user"
+import { isUrlValid } from "@server/helpers/custom-validators/activitypub/misc"
+
+type AvailableForListOptions = {
+  accountId: number
+  search?: string
+  host?: string
+  handles?: string[]
+  forCount?: boolean
+}
+
+@DefaultScope(() => ({
+  include: [
+    {
+      model: VideoChannelModel, // Default scope includes avatar and server
+      required: true
+    }
+  ]
+}))
+@Table({
+  tableName: 'videoChannelSync',
+  indexes: [
+    {
+      fields: [ 'videoChannelId' ]
+    }
+  ]
+})
+export class VideoChannelSyncModel extends Model<Partial<AttributesOnly<VideoChannelSyncModel>>> {
+
+  @AllowNull(false)
+  @Default(null)
+  @Is('VideoChannelExternalChannelUrl', value => throwIfNotValid(value, isUrlValid, 'externalChannelUrl', true))
+  @Column(DataType.STRING(CONSTRAINTS_FIELDS.VIDEO_CHANNEL_SYNCS.EXTERNAL_CHANNEL_URL.max))
+  externalChannelUrl: string
+
+  @CreatedAt
+  createdAt: Date
+
+  @UpdatedAt
+  updatedAt: Date
+
+  @ForeignKey(() => VideoChannelModel)
+  @Column
+  videoChannelId: number
+
+  @BelongsTo(() => VideoChannelModel, {
+    foreignKey: {
+      allowNull: false
+    },
+    onDelete: 'cascade'
+  })
+  VideoChannel: VideoChannelModel
+
+  @AllowNull(false)
+  @Default(VideoChannelSyncState.WAITING_FIRST_RUN)
+  @Is('VideoChannelSyncState', value => throwIfNotValid(value, isVideoChannelSyncStateValid, 'state'))
+  @Column
+  state: VideoChannelSyncState
+
+  @AllowNull(true)
+  @Column(DataType.DATE)
+  lastSyncAt?: Date
+
+  static listByAccountForAPI (options: Pick<AvailableForListOptions, 'accountId'> & {
+    start: number
+    count: number
+    sort: string
+  }) {
+    const query = {
+      offset: options.start,
+      limit: options.count,
+      order: getSort(options.sort),
+      include: [
+        {
+          model: VideoChannelModel,
+          required: true,
+          where: {
+            accountId: options.accountId
+          }
+        }
+      ]
+    }
+    return Promise.all([
+      VideoChannelSyncModel.count(query),
```

Comment:
`count` should use `VideoChannelModel.unscoped` include or the count may be wrong because of channel multiple avatars
`VideoChannelSyncModel` should also be `unscoped()` to prevent double `VideoChannelModel` include